crates/forge/tests/integration/resources.rs

```diff
@@ -193,3 +193,42 @@ fn accumulate_syscalls() {
     assert_syscall(&result, "single_write", StorageWrite, 1);
     assert_syscall(&result, "double_write", StorageWrite, 2);
 }
+
+#[test]
+fn steps_count_includes_os_resources() {
```

Comment:
I guess then this test doesn't have anything to do with steps?

Reply:
right, corrected